frontend-pwa/src/app/App.tsx

```
@@ -13,25 +13,17 @@ export function App() {
 
   if (isLoading) {
     return (
-      <p
-        className="p-6 min-h-screen bg-base-200 text-base-content"
-        role="status"
-        aria-live="polite"
-      >
+      <div className="p-6 min-h-screen bg-base-200 text-base-content" role="status">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)